### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Koliras**
- [DEMO LINK](https://Koliras.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Koliras.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img src="./images/logo.png" alt="MOYO logo"
+          class="logo">
```
Comment (BudnikOleksii) [inline src/index.html:15]: [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 3 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,42 @@
...
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover"
+            >Laptops & Computers</a>
```
Comment (BudnikOleksii) [inline src/index.html:32]: The same
image

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,71 @@
...
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  font-family: "Roboto", "Helvetica", sans-serif;
```
Comment (BudnikOleksii) [inline src/style.css:34]: The main font-family should be specified for html or body element

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Good job👍
Left some comments, lets fix a bit.

File: src/index.html [hunk 4]
```diff
@@ -9,6 +9,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+          class="logo">
```
Comment (Moroz-Dmytro) [inline src/index.html:17]:
```suggestion
        <img
          src="./images/logo.png"
          alt="MOYO logo"
          class="logo"
        >
```

File: src/index.html [hunk 5]
```diff
@@ -9,6 +9,45 @@
...
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```
Comment (Moroz-Dmytro) [inline src/index.html:24]:
```suggestion
          </li>
          
          <li class="nav__item">
```

Comment (Moroz-Dmytro) [reply src/index.html:24]: And in all other places

File: src/index.html [hunk 6]
```diff
@@ -9,6 +9,45 @@
...
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover"
+            >Laptops & Computers</a>
```
Comment (Moroz-Dmytro) [inline src/index.html:35]:
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,71 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --blue: #00acdc;
```
Comment (Moroz-Dmytro) [inline src/style.css:4]:
```suggestion
  --blue-color: #00acdc;
```

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Good work 👍 
But let's fix this on smaller screens
image

File: src/style.css (hunk 7 above)
Comment (BudnikOleksii) [reply src/style.css:4]: Don't forget to check all comments.

**Review by BudnikOleksii — APPROVED** [review]
Good job!
Don't forget to check all comments.

**Pull request closed without merging**